bin/informational/adobe-id-for-short-client-token

```diff
@@ -8,7 +8,7 @@ import base64
 import re
 import sys
 
-base_url = "https://circulation.librarysimplified.org"
+base_url = "https://libraryregistry.librarysimplified.org"
```

Comment:
An unrelated change as long as I was messing around with scripts.

circulation.librarysimplified.org no longer manages the Short Client Token. This script wasn't broken, but it was going to the wrong server.